rwilliams659/overlook-app
# Rw manager dash post delete

#### What changes/additions are a part of this PR?
This gets the POST and DELETE requests working, allowing a manger to add and delete reservations for a user! I also fetch new booking data to update it after either of the above events. 

#### Any background context you want to provide?
Everything above has been tested to ensure data on server is updating correctly. 

#### Where should the reviewer start?
Index.js

#### How should this be tested (test suite/manually)?
No test suite exists yet - test this via the app and console, where you can see any errors/status codes with the fetch API

#### What are the relevant tickets?
Ticket #37
